tests/state_test.py

```diff
@@ -831,7 +831,7 @@ class StateHypothesisTest(jtu.JaxTestCase):
 
     @hp.given(get_vmap_params())
     @hp.settings(deadline=None, print_blob=True,
-        max_examples=config.FLAGS.jax_num_generated_cases)
+        max_examples=jtu._NUM_GENERATED_CASES.value)
```

Comment:
Please don't use underscore-prefixed values outside their enclosing module. Remove the underscore prefix?

Reply:
Ooops, missed this somehow. Will fix.